src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjectsTable.tsx

```diff
@@ -1,93 +1,114 @@
 import { Box } from '@mui/material';
-import { DataGrid, DataGridProps, GridRowParams } from '@mui/x-data-grid';
+import {
+  DataGrid,
+  DataGridProps,
+  GridColDef,
+  GridRowParams,
+} from '@mui/x-data-grid';
 import { cmpBy, simpleSwitch } from '@seedcompany/common';
 import { useNavigate } from 'react-router-dom';
 import { ProjectStatusLabels } from '~/api/schema/enumLists';
 import { Sensitivity } from '~/api/schema/schema.graphql';
 import { labelFrom } from '~/common';
 import { SensitivityIcon } from '~/components/Sensitivity';
-import { PartnerDetailProjectsTableListItemFragment as Project } from './PartnerDetailProjectsTable.graphql';
+import {
+  PartnerDetailProjectsTableListItemFragment,
+  PartnerDetailProjectsTableListItemFragment as Project,
+} from './PartnerDetailProjectsTable.graphql';
 
 export interface PartnerDetailProjectsTableProps {
-  projects: readonly Project[];
-  GridProps?: Omit<DataGridProps<Project>, 'columns' | 'isCellEditable'>;
+  projects: readonly PartnerDetailProjectsTableListItemFragment[];
+  onPageChange?: (page: number) => void;
+  totalRows?: number;
+  GridProps?: Omit<
+    DataGridProps<Project>,
+    'columns' | 'rows' | 'isCellEditable'
+  >;
 }
```

Comment:
This component is really just a configured `DataGrid`, so I would expose the `DataGrid` props directly.

Having the prop names consistent reduces cognitive complexity too.
projects -> rows
totalRows -> rowCount

```ts
export interface PartnerDetailProjectsTableProps
  extends Omit<DataGridProps<Project>, 'columns'> {}
```

Reply:
I don't quite understand this one, unfortunately. Could you explain what you mean by "expose directly"

Reply:
I just mean expose the GridProps directly instead of under a GridProps key. Since this component is a decorated Grid, there's nothing really else it could be.